src/test/java/uk/gov/justice/digital/delius/integration/secure/UserAPITest.java

```diff
@@ -62,21 +60,26 @@ public void retrieveUserDetailsForMultipleUsers() throws JsonProcessingException
                 .statusCode(200)
                 .extract()
                 .body()
-                .as(JsonNode.class);
-
-        Map<String, UserDetails> userDetailsMap = objectMapper.convertValue(userDetails, new TypeReference<>() {});
-        UserDetails jimSnowUserDetails = userDetailsMap.get("JimSnowLdap");
-        UserDetails sheilaHancockUserDetails = userDetailsMap.get("sheilahancocknps");
-
-        assertThat(userDetails.size()).isEqualTo(2);
-
-        assertThat(jimSnowUserDetails.getEmail()).isEqualTo("jim.snow@justice.gov.uk");
-        assertThat(jimSnowUserDetails.getFirstName()).isEqualTo("Jim");
-        assertThat(jimSnowUserDetails.getSurname()).isEqualTo("Snow");
-
-        assertThat(sheilaHancockUserDetails.getEmail()).isEqualTo("sheila.hancock@justice.gov.uk");
-        assertThat(sheilaHancockUserDetails.getFirstName()).isEqualTo("Sheila");
-        assertThat(sheilaHancockUserDetails.getSurname()).isEqualTo("Hancock");
+                .as(JsonNode.class)
+                .toString();
+
+        String expectedJson = "{" +
+                        "\"userDetailsList\":[{" +
+                           "\"firstName\":\"Jim\"," +
+                           "\"surname\":\"Snow\"," +
+                           "\"email\":\"jim.snow@justice.gov.uk\"," +
+                           "\"enabled\":true," +
+                           "\"username\":\"JimSnowLdap\"" +
+                       "},{" +
+                           "\"firstName\":\"Sheila\"," +
+                           "\"surname\":\"Hancock\"," +
+                           "\"email\":\"sheila.hancock@justice.gov.uk\"," +
+                           "\"enabled\":true," +
+                           "\"username\":\"sheilahancocknps\"" +
+                       "}]" +
+                       "}";
+
+        assertThat(userDetails).isEqualTo(expectedJson);
```

Comment:
I was about to write a comment about how you can't guarantee the format of the json, but then realised that you're converting into a `JsonNode` and then converting that to a string so the format will be guaranteed :grinning: .  One alternative to this is to use jsonassert or another json library to compare the two strings...